gloo-edge/gloo-edge/README.md

```diff
@@ -1372,41 +2224,256 @@ spec:
   sslConfig:
     secretRef:
       name: upstream-tls
-      namespace: gloo-system
+      namespace: gloo-system  
   virtualHost:
-    options:
-      extauth:
-        configRef:
-          name: keycloak-oauth
-          namespace: gloo-system
     domains:
       - '*'
+    options:
+      ratelimitBasic:
+        anonymousLimits:
+          requestsPerUnit: 5
+          unit: MINUTE
+        authorizedLimits:
+          requestsPerUnit: 20
+          unit: MINUTE
+      waf:
+        customInterventionMessage: "Blocked Scammer"
+        ruleSets:
+        - ruleStr: |
+            SecRuleEngine On
+            SecRule REQUEST_HEADERS:User-Agent "scammer" "deny,status:403,id:107,phase:1,msg:'blocked scammer'"
+      stagedTransformations:
```

Comment:
We don't want to use this transformation anymore because there's now an `idTokenHeader` option in the `AuthConfig`

Reply:
done